guides/source/initialization.md

```diff
@@ -464,7 +464,7 @@ The `options[:config]` value defaults to `config.ru` which contains this:
 ```ruby
 # This file is used by Rack-based servers to start the application.
 
-require ::File.expand_path('../config/environment', __FILE__)
+require_relative 'config/environment'
```

Comment:
We are requiring from the previous directory `..`, not the current one, this is not the same as what is already mentioned here.